### DISCUSSION TITLE
自己作品上傳到NFT會不會被告

### DISCUSSION THREAD

**Discussion by HuanYuuuuuu** [1101-GS4719-C/Python-Community-GS4719-C#442] [Q&A] [answered]
NFT，又叫做非同質化代幣（Non-fungible token，簡稱「NFTs」），目前主要是應用在電子藝術品上面，可以證明買家確實擁有與特定數位藝術品、音樂或其他作品相連結的無形標記。有很多有錢的投資人會花大筆的金錢購買這些虛擬藝術品。但與其說是購買這些藝術品，比較像是購買這些藝術品的專利。 但其實我對他們放上的照片的所有權的部分其實感覺蠻模糊的
像我想說我也可以把一些東西放上NFT的平台上，於是我把女生服飾店裡給男友做的椅子的照片都放上Open sea (NFT的平台之一)。
但我其實不知道這樣做會不會涉及那些椅子設計師的權利問題等等
之後也想把自己無聊時間畫的畫上傳到NFT平台上，但有些是看著別人的畫風跟著學習的，不知道會不會有抄襲、二創、模仿等等之類的問題
![未命名](https://user-images.githubusercontent.com/99785986/157386836-b0b61936-511d-485a-829c-612c5ec9c20c.png)

**Comment by chia-kai-chang** [comment, marked as answer]
藝術品沒有專利。創作是要看著作權。著作權是國家給予的法律權利，由國家的法律保護。

NFT只是代表一段網址而已。沒有著作權跟版權的概念。網路上的youtuber大部分是過度解釋了。

> 像我想說我也可以把一些東西放上NFT的平台上，於是我把女生服飾店裡給男友做的椅子的照片都放上Open sea (NFT的平台之一)。
但我其實不知道這樣做會不會涉及那些椅子設計師的權利問題等等

如果照片是你拍攝的，你就有這張照片的**著作權**。如果照片不是你拍攝的，你放到Opensea販售，目前無法可管。(如有錯誤，請指正)

> 之後也想把自己無聊時間畫的畫上傳到NFT平台上，但有些是看著別人的畫風跟著學習的，不知道會不會有抄襲、二創、模仿等等之類的問題

**重新創作**(二創)就是你自己的著作權。「自己創作」或「抄襲模仿」由國家法律機關評斷。但NFT平台不管這個，如果NFT商品營利，賣的標的物是網址，而非創作作品本身。

**Comment by jason79461385** [comment]
記得之前統神兒子的照片就有被做成nft，但因為沒經過統神他們同意所以被要求下架了。
如果沒有侵犯到別人肖像權我想是不用擔心
